packages/api/cms-api/src/mikro-orm/migrations/Migration20231206123505.ts

```diff
@@ -0,0 +1,13 @@
+import { Migration } from "@mikro-orm/migrations";
+
+export class Migration20231206123505 extends Migration {
+    async up(): Promise<void> {
+        const result = await this.execute(`select count(*) from "Migrations" where name = 'Migration20230831110518';`);
```

Comment:
In theory there could be projects which have a different migrations table name. Should be take that into consideration? It would be possible using `const migrationsTableName = this.config.get("migrations").tableName`

Reply:
f1183e3eed4f1b8b0afe593889f2dc9fb609b7d9